lib/src/event_dispatcher.dart

```diff
@@ -130,7 +131,15 @@ class EventDispatcher implements IEventDispatcher {
         break;
 
       case "playerUpdate":
-        onPlayerUpdateController.add(PlayerUpdateEvent(cluster.client, node, json["data"] as Map<String, dynamic>));
+        final update = PlayerUpdateEvent(cluster.client, node, json["data"] as Map<String, dynamic>);
+
+        if (update.state.position != null) {
+          final _node = node as Node;
+          // Update the position of the currently playing track of the corresponding player.
+          (_node.players[update.guildId]?.nowPlaying?.track.info as TrackInfo).position = update.state.position!;
```

Comment:
Should the cast here be nullable, or are we certain that all the properties will be non-null?

Reply:
yeah, it should be nullable, i didn't notice about that, i'm changing it